<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emp_verification</class>
 <widget class="QDialog" name="emp_verification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verification</string>
  </property>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>4</width>
     <height>12</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>207</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_Username"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_Password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="signInButton">
      <property name="text">
       <string>Sign in</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
